treeherder/webapp/api/projects.py

```diff
@@ -6,15 +6,16 @@
 from treeherder.model.derived import DatasetNotFoundError
 import simplejson as json
 
-from treeherder.model.derived import JobsModel
+from treeherder.model.derived import JobsModel, ArtifactsModel
 
 
 def project_info(request, project):
     try:
-        jm = JobsModel(project)
-        return HttpResponse(content=json.dumps({'max_job_id': jm.get_max_job_id(),
+        jobs_model = JobsModel(project)
+        artifacts_model = ArtifactsModel(project)
```

Comment:
maybe you can use a context manager here to make sure the connection is eventually closed?